ts/features/fci/screens/FciRouterScreen.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { constNull } from "fp-ts/lib/function";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { SignatureRequestDetailView } from "../../../../definitions/fci/SignatureRequestDetailView";
+import { IOStackNavigationRouteProps } from "../../../navigation/params/AppParamsList";
+import { useIODispatch, useIOSelector } from "../../../store/hooks";
+import { FciParamsList } from "../navigation/params";
+import { fciSignatureRequestFromId } from "../store/actions";
+import { fciSignatureRequestSelector } from "../store/reducers/fciSignatureRequest";
+import { LoadingErrorComponent } from "../../bonus/bonusVacanze/components/loadingErrorScreen/LoadingErrorComponent";
+import SuccessComponent from "../components/SuccessComponent";
+import GenericErrorComponent from "../components/GenericErrorComponent";
+
+export type FciRouterScreenNavigationParams = Readonly<{
+  signatureRequestId: SignatureRequestDetailView["id"];
+}>;
+
+const FciSignatureScreen = (
+  props: IOStackNavigationRouteProps<FciParamsList, "FCI_ROUTER">
+): React.ReactElement => {
+  const dispatch = useIODispatch();
+  const fciSignatureRequest = useIOSelector(fciSignatureRequestSelector);
+
+  React.useEffect(() => {
+    const signatureRequestId = props.route.params.signatureRequestId;
```

Comment:
What you think about creating a const with signatureRequestId and than use it here and in the dependency array?